### ISSUE TITLE
Is there a way to craft a weighted-per-batch MSE loss function?

### ISSUE THREAD

**Issue by joseabernal** [keras-team/keras#9716]
Dear all,

Is there a way to create an MSE loss weighted per batch? The approach would not follow https://github.com/keras-team/keras/issues/2115 since each batch would be weighted differently. 

Best regards,
